src/components/header/_header.scss

```diff
@@ -24,9 +24,13 @@
 
   }
 
-  .govuk-header--full-width {
+  .govuk-header__container--full-width {
```

Comment:
I find it a _bit_ weird that the _container_ class adds a border and has a side effect on a contained element – I'd expect the container just to set margins and width… but might just be me. Anyone else got any thoughts on the matter?

Reply:
Yep, it bugs me too, but otherwise I'll have to check the classes applied to the header and the ones applied to the container to make sure we don't have 'strange' mix (e.g. `classes: govuk-header--full-width` and `containerClasses: govuk-width-container`)

Reply:
I'd go with what we have now, and reconsider modifiers again in the future as it allows you to do what you need to inside the component.

Reply:
👍